bptestrunner/bluepill_batch_test_runner.template.sh

```diff
@@ -41,6 +42,9 @@ for test_host in ${TEST_HOST_PATHS[@]}; do
         TEST_HOST_NAME=$(basename_without_extension "${test_host}")
         unzip -qq -d "$BP_WORKING_FOLDER" "$test_host"
         cp -cr "${BP_WORKING_FOLDER}/Payload/${TEST_HOST_NAME}.app" ${BP_WORKING_FOLDER}
+    elif [[ $test_host == *.app ]]; then
+        cp -cr $test_host $BP_WORKING_FOLDER
+        chmod -R 777 "$BP_WORKING_FOLDER/$(basename "$test_host")"
```

Comment:
Here are the permissions I was seeing without modification:
```
dr-xr-xr-x  SlackFoundationTests.xctest
dr-xr-xr-x  SlackUnitTestHost.app
```
Which caused this error:
```
{55961} 20210804.150615 [  ERROR ] (BP-1) Completed: [Attempt 1] Install Application
{55961} 20210804.150615 [  ERROR ] (BP-1) Could not install app in simulator: copyfile of file:///var/tmp/_bazel_jschear/43dcaece8ddcfdb7625d810085488313/execroot/slack-objc/_tmp/711a7c3713c5c98515033962dccc4351/bp_exec_root/SlackUnitTestHost.app/ to file:///Users/jschear/Library/Developer/CoreSimulator/Devices/538FFAD4-D306-4E62-84CB-94A667A64BD7/data/Containers/Shared/SystemGroup/systemgroup.com.apple.installcoordinationd/Library/InstallCoordination/PromiseStaging/B8F3E7F8-1895-4B6F-B670-6F298AA4043A/SlackUnitTestHost.app failed: Permission denied
```

Reply:
Ditto here

```suggestion
        chmod -R ug+w "$BP_WORKING_FOLDER/$(basename "$test_host")"
```